Extensions/auto_tagger.js

```diff
@@ -289,6 +301,18 @@ XKit.extensions.auto_tagger = new Object({
 
 		}
 
+
+		if (XKit.extensions.auto_tagger.preferences.tag_source.value && obj.source_owner !== "") {
```

Comment:
suggestion: use `!obj.source_owner` to encompass both the `!== undefined` and `!== ""` checks (see below)